src/app/pages/market/ProductAction.php

```diff
@@ -35,40 +36,68 @@ public function __invoke($request, $response, $args)
     $installNow = isset($request->getQueryParams()['installNow']);
     $mavenProductInfo = $product->getMavenProductInfo();
     $version = $args['version'] ?? '';
+    $initVersion = $args['version'] ?? '';
     $mavenArtifactsAsDependency = [];
     $mavenArtifacts = [];
     $docUrl = '';
+    $versionsToDisplay = null;
+
+    $showDevVersionCookie = $request->getCookieParams()['showDevVersions'] ?? 'false';
+    $showDevVersions = filter_var($showDevVersionCookie, FILTER_VALIDATE_BOOLEAN) ?? false;
+    if (isset($request->getQueryParams()['showDevVersions'])) {
+      if ($request->getQueryParams()['showDevVersions'] == "true") {
+        $showDevVersions = true;
+        setcookie('showDevVersions', "true", time()+60*60*24*30 , "/");
```

Comment:
May be even higher? Or do you renew it?

Reply:
uups copied. 30days. no I don't renew it. should be higer. yes.